src/components/Box/index.tsx

```diff
@@ -30,7 +30,7 @@ export type BoxProps = BackgroundSetProps &
   LayoutSetProps &
   TransformSetProps;
 
-const Box = styled(polymorph<BoxProps>("div"))(
+const Box = styled(polymorph<BoxProps>("div"))<BoxProps>(
```

Comment:
Was this causing the type check to fail otherwise?

Reply:
Yeah